slynk/slynk-backend.lisp

```diff
@@ -1487,9 +1487,11 @@ Return :interrupt if an interrupt occurs while waiting."
      (error
       "~s not implemented. Check if ~s = ~s is supported by the implementation."
       'wait-for-input
-      (slynk-backend:find-symbol2 "SLYNK:*COMMUNICATION-STYLE*")
+      (slynk-backend:find-symbol2 #1=#.(if (eq :UPCASE (readtable-case *readtable*))
```

Comment:
Can't we hide whatever logic we need to hide inside `SLYNK-BACKEND:FIND-SYMBOL2`?